docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -1612,8 +1591,74 @@ Project configuration example:
 
 The above example will be used to define the following arguments when running the test:
 
+[source,shell]
 ----
 --int-arg PIXIT.ACE.APPENDPOINT:1 PIXIT.ACE.APPDEVTYPEID:256 --string-arg PIXIT.ACE.APPCLUSTER:OnOff PIXIT.ACE.APPATTRIBUTE:OnOff
 ----
 
+====== Test Parameters Examples
 
+Access the spreadsheet via the https://groups.csa-iot.org/wg/matter-csg/document/36336[Verification Steps Document] and review the information provided. Based on this data, create the parameters set as requested. 
+
+Below are some specific examples assembled from data obtained from the spreadsheet.
+
+. TC-ACE-1.4
++
+Sample command to run manually inside docker
++
+[source,shell]
+----
+python3 TC_ACE_1_4.py --discriminator 3840 --passcode 20202021 --commissioning-method on-network --storage-path admin_storage.json  --int-arg PIXIT.ACE.APPENDPOINT:1 PIXIT.ACE.APPDEVTYPEID:256 --string-arg PIXIT.ACE.APPCLUSTER:OnOff PIXIT.ACE.APPATTRIBUTE:OnOff  --paa-trust-store-path /credentials/development/paa-root-certs/
```

Comment:
@hiltonlima did you test the sample scripts to validate the translation from VS to TH? I'm asking because one of the complains from users is sometimes the mapping between command-line and TH project config values is not direct and we need to gradually identify and correct those cases.

Reply:
These examples were taken AS IS from the step verification spreadsheet.

I have not run them in my environment.

Reply:
<img width="1610" alt="image" src="https://github.com/user-attachments/assets/1d85fd57-75b7-42bc-867c-de52ebc1503c">